src/bootstrap/bin/rustc.rs

```diff
@@ -107,7 +107,9 @@ fn main() {
     if let Some(target) = target {
         // The stage0 compiler has a special sysroot distinct from what we
         // actually downloaded, so we just always pass the `--sysroot` option.
```

Comment:
This comment I think is no longer correct

Reply:
I updated it.